WhiteDentalClinic/WhiteDentalClinic.DataAccess/Repositories/CustomerRepository/CustomerRepository.cs

```diff
@@ -0,0 +1,38 @@
+﻿using WhiteDentalClinic.DataAccess.Entities.CustomerEntity;
+
+namespace WhiteDentalClinic.DataAccess.Repositories.CustomerRepository
+{
+    public class CustomerRepository : BaseRepository<Customer>, ICustomerRepository
+    {
+        public CustomerRepository(ApiDbTempContext context) : base(context)
+        { }
+
+        /* public List<Customer> GetCustomers()
```

Comment:
Not needed comment. You already have all this functionality in the base repository.